File: benchmark/PHP 7.0/字符串追加性能测试.php
<?php

$t = microtime(true);

for ($i = 0; $i < 1000000; ++$i)
{
	$s = "";
	$s .= "foo";
	$s .= "bar";
	$s .= "ok";
	$s .= $i;
}

echo (microtime(true) - $t), "\n";
echo $s, "\n";

File: benchmark/Lua 5.3/字符串追加性能测试.lua
local start = os.clock()

local s
local a = {}
for i = 0, 1000000-1, 1 do
	a[0] = "foo"
	a[1] = "bar"
	a[2] = "ok"
	a[3] = i
	s = table.concat(a)
end

print(os.clock()-start, "\n")
print(s, "\n");
